themes/question/image_select/survey/questions/answer/listradio/config.xml

```diff
@@ -80,7 +80,6 @@
             <category>Display</category>
             <sortorder>90</sortorder>
             <inputtype>integer</inputtype>
-            <default>-1</default>
```

Comment:
Not a fan of this change, since for a code reader it will look like default tag is forgotten. Will it work with `<default></default>`?

Reply:
No : it never work with `<default></default>` , issue was reported

Reply:
_Should_ it work?

Reply:
I think : it should work, not necesssary if we use `QuestionAttribute::getDefaultSettings();` but someone remove the usage of this function done to give the default in 4 … previously

Reply:
Sigh.